backend-nodejs/.sequelizerc

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```

Comment:
It's probably helpful to eliminate all things _sequelize_ from the pull request.  I doubt that the Trash API app will require any persistence (beyond committing images to an S3 bucket).  All of the _sequelize_ artifacts will just continue to clutter things up.